R/variable-derivation.R

```diff
@@ -0,0 +1,124 @@
+#' Get or set a derived variable's `CrunchExpr`
```

Comment:
just say "expression" here and not CrunchExpr. And I think that the first line of docs like this is the title and thus should be Title Cased (yes, I know the current docs aren't consistent about that)